Jenkinsfile

```diff
@@ -3,10 +3,9 @@
 buildPlugin(useContainerAgent: true,
         configurations: [
                 [platform: 'linux', jdk: '11'],
-                [platform: 'windows', jdk: '8'],
 
                 // testing the Guava & Guice bumps
                 // https://github.com/jenkinsci/jenkins/pull/5707
                 // https://github.com/jenkinsci/jenkins/pull/5858
```

Comment:
Sorry that I didn't comment on this earlier, but these comments are no longer valid.  We merged the Guava and Guice changes a long time ago.
```suggestion
```